### ISSUE TITLE
[Feature request] Publish the 2M-step checkpoint

### ISSUE THREAD

**Issue by PavlosMelissinos** [KranthiGV/Pretrained-Show-and-Tell-model#1]
Hi and thanks for this!

Could you maybe also convert the second checkpoint [from this comment](https://github.com/tensorflow/models/issues/466#issuecomment-251756098) and upload it?

And maybe [this one (after 3M steps)](https://github.com/tensorflow/models/issues/466#issuecomment-271822868).

**Comment by KranthiGV** [comment]
Hello @PavlosMelissinos,
I'm afraid there's no way to convert the previous checkpoints created by other users using a different version of the tokenizer. The model is specific to the `word_counts.txt` file generated.
Nevertheless, I am currently training to create the 2M  checkpoint.
I will upload it in a few days.
Thanks!

**Comment by PavlosMelissinos** [comment]
Oh I see. I somehow misread your comments on the other thread and thought you did the conversion. I will be waiting for the update then, thanks.

**Comment by KranthiGV** [comment]
@PavlosMelissinos 
Greetings!
The 2M finetuned version is uploaded now.

**Comment by vgoklani** [comment]
Thank you! I'm just curious, have you seen any improvements in the quality of the captions? How are you measuring performance?

**Comment by KranthiGV** [comment]
@vgoklani 
I have not yet evaluated the model.
But, finetuning would certainly improve the performance of the model.
## Quantitatively:
You can verify it by evaluating the model using a metric such as BLEU score or CIDER.
## Qualitatively:
In the initial training the LSTM part of the network updates its parameters.
When we finetune, we allow the CNN part of the network to identify the objects of interest (by **updating its parameters**) in a way more suitable to generate better captions (Unlike in pure CNN part where the goal is object detection, now the goal is to generate better captions which can be seen in the loss function defined)
